main.py

```diff
@@ -412,11 +430,7 @@ def attemptBountyFill(bot, waifuid):
                             "A higher bounty for [%d] %s than yours was filled, so you can now cancel yours and get full points back provided you don't change it." % (
                                 waifuid, order[4]), True)
             # give the disenchanter appropriate profit
-            base_value = int(config["rarity" + str(order[5]) + "Value"])
-            if order[3] > rarity_cap:
-                return (order[3] - rarity_cap) // 4 + (rarity_cap - base_value) // 2
-            else:
-                return max(math.floor((order[3] - base_value) * 0.5), 2)
+            return max((order[3] - rarity_cap) // 4, 0) + max(min(order[3] - min_bounty, rarity_cap - min_bounty) // 2, 0) + (min_bounty - base_value)
```

Comment:
One-Liners are nice and short, but i feel i couldn't parse this as easily; A Comment explaining it is probably a good idea